pkg/assets/manifests/operators.go

```diff
@@ -0,0 +1,115 @@
+package manifests
+
+import (
+	"path/filepath"
+
+	"github.com/ghodss/yaml"
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/pkg/errors"
+	v1core "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	manifestDir = "manifests"
+)
+
+var (
+	aroOperatorNamespaceFilename = filepath.Join(manifestDir, "aro-operator-00.yml")
+	aroOperatorFilename          = filepath.Join(manifestDir, "aro-operator-01.yml")
+)
+
+// AroOperator generates the Aro Operator files.
+type AroOperator struct {
+	FileList []*asset.File
+}
+
+var _ asset.WritableAsset = (*AroOperator)(nil)
+
+// Name returns a human friendly name for the asset.
+func (*AroOperator) Name() string {
+	return "AroOperator Config"
+}
+
+// Dependencies returns all of the dependencies directly needed to generate
+// the asset.
+func (*AroOperator) Dependencies() []asset.Asset {
+	return []asset.Asset{}
```

Comment:
shouldn't this have the other openshift manifests as a dependancy? other wise it might start this before the cluster?